kv/service.go

```diff
@@ -49,15 +48,13 @@ type Service struct {
 	endpointStore *IndexStore
 	variableStore *IndexStore
 
-	Migrator *Migrator
-
 	urmByUserIndex *Index
 
 	disableAuthorizationsForMaxPermissions func(context.Context) bool
 }
 
 // NewService returns an instance of a Service.
-func NewService(log *zap.Logger, kv Store, configs ...ServiceConfig) *Service {
+func NewService(log *zap.Logger, kv SchemaStore, configs ...ServiceConfig) *Service {
```

Comment:
note to self: demote to `kv.Store` (schema store only useful for migrations now)